<launch>
  <node pkg="drix_project11" type="robobox_interface" name="robobox_interface" ns="project11"/>
</launch>
